src/ontology/mondo-edit.obo

```diff
@@ -133500,33 +133500,11 @@ relationship: excluded_subClassOf MONDO:0003686 {source="DOID:3893", source="htt
 
 [Term]
 id: MONDO:0006788
-name: hydrophthalmos
-def: "Abnormal enlargement of the eye" [NCIT:P378]
-comment: Reason: duplicate. This will be merged with MONDO:0020366 congenital glaucoma
-subset: gard_rare {source="MONDO:GARD"}
-subset: obsoletion_candidate
-subset: rare
-xref: DOID:11212 {source="MONDO:equivalentTo", source="EFO:1000968"}
-xref: ICD10CM:Q15.0 {source="DOID:11212"}
-xref: ICD9:743.2 {source="DOID:11212"}
-xref: ICD9:743.20 {source="DOID:11212"}
-xref: MESH:D006871 {source="MONDO:equivalentTo", source="EFO:1000968", source="DOID:11212"}
-xref: NCIT:C50648 {source="DOID:11212"}
-xref: SCTID:13832009 {source="DOID:11212"}
-xref: SCTID:156898000 {source="DOID:11212"}
-xref: SCTID:204111004 {source="DOID:11212"}
-xref: SCTID:204112006 {source="DOID:11212"}
-xref: SCTID:204113001 {source="DOID:11212"}
-xref: SCTID:204114007 {source="DOID:11212"}
-xref: SCTID:204115008 {source="DOID:11212"}
-xref: SCTID:204121007 {source="DOID:11212"}
-xref: SCTID:392368005 {source="DOID:11212"}
-xref: SCTID:392371002 {source="DOID:11212"}
-xref: SCTID:413728006 {source="DOID:11212"}
-is_a: MONDO:0000365 {source="DOID:11212"} ! primary congenital glaucoma
-is_a: MONDO:0020366 {source="EFO:1000968", source="MONDO:Redundant", source="MONDO:indirect"} ! congenital glaucoma
```

Comment:
Was there a reason to not include these SubClassOf axioms to MONDO:0006789?

Reply:
Yes, the term was merged with `congenital glaucoma` so I removed this `is_a` so it's not stated to be a subClassOf itself. 

The `is_a` for `primary congenital glaucoma` was removed because this would cause a cycle of `congenital glaucoma`--> 'primary congenital glaucoma` (`is_a` from merge) --> `congenital glaucoma`.